spec/components/candidate_interface/application_complete_content_component_spec.rb

```diff
@@ -10,11 +10,6 @@
     end
   end
 
-  before do
-    view_helper = instance_double(ViewHelper)
-    allow(view_helper).to receive(:respond_by_date).and_return('1 January 2020')
-  end
```

Comment:
yea this can go now since it's logic encapsulated in the component. No need to stub any requests 👍